hpcbench/driver.py

```diff
@@ -238,17 +232,23 @@ def nodes(self, tag):
 class CampaignDriver(Enumerator):
     """Abstract representation of an entire campaign"""
     def __init__(self, campaign_file=None, campaign_path=None,
+                 campaign=None,
                  node=None, output_dir=None, srun=None,
                  logger=None, expandcampvars=True):
         node = node or socket.gethostname()
-        if campaign_file and campaign_path:
-            raise Exception('Either campaign_file xor path can be specified')
+        if campaign_file and campaign_path and campaign:
```

Comment:
if two of the three operands evaluate to True and one evaluates to False, the expression will still evaluate to False - so, I think this condition is not quite right

Reply:
You're right!